examples/karaf-rest-example/karaf-rest-example-features/src/main/feature/feature.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<features name="karaf-rest-example-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.3.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
+
+    <repository>mvn:org.apache.cxf.karaf/apache-cxf/3.2.1/xml/features</repository>
+    
+    <feature name="karaf-rest-example-common" version="${project.version}">
+        <feature dependency="true">http</feature>
+        <requirement>osgi.service;effective:=active;filter:=(objectClass=org.osgi.service.http.HttpService)</requirement>
+        <feature version="[3,4)">cxf-jaxrs</feature>
+        <feature version="${cxf.version}">cxf-jaxrs</feature>
```

Comment:
There's two `<feature/>` definition for `cxf-jaxrs`. I would remove the later to use the one with the version range.

Reply:
It's a mistake from me I forgot to clean, I remove the later.

Reply:
Can I put this in pom parent properties and use it in feature.xml ?
```
<cxf.version>[3,4)</cxf.version>
```